ADOPTERS.md

```diff
@@ -1,9 +1,10 @@
 # Adopters of Kubeflow Pipelines
 
-Below are the adopters of project Pipelines. If you are using Pipelines
+Below are the adopters of project Kubeflow Pipelines. If you are using Kubeflow Pipelines
 please add yourself into the following list by a pull request.
 Please keep the list in alphabetical order.
 
-| Organization                                     | Contact                                                           | Description of Use                                     |
-|--------------------------------------------------|-------------------------------------------------------------------|--------------------------------------------------------|
-| [Red Hat](https://www.redhat.com/)               | [@franciscojavierarceo](https://github.com/franciscojavierarceo)  | ML/AI & Data orchestration  |
+| Organization                                                                 | Contact                                                                                           | Description of Use                                     |
+|------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|--------------------------------------------------------|
+| [IBM Research Foundation Model Data Engineering Team](https://www.research.ibm.com/) | [@roytman](https://github.com/roytman), [@yuanchi2807](https://github.com/yuanchi2807)           | Foundation Model Data Engineering                       |
```

Comment:
Hi, I suggest replacing our name's order. Yuan Chi is from the same team, and he will be more up to date with what's going on.

Reply:
Updated!